### ISSUE TITLE
Allow for checks made inline on terminal using the | method

### ISSUE THREAD

**Issue by cjavad** [cjavad/hashit#5]
Allow for hashit to be used like shasum and such, so you can take check input like this when no file is specified:

```sh
echo "add4614b6fe3bb8e7dddcaab0ea97c476fbd4ffe288f2a4912cb06f1a47dcfa0 *ubuntu-18.04.3-desktop-amd64.iso" | shasum -a 256 --check
```

With hashit it should look like this:
```sh
echo "add4614b6fe3bb8e7dddcaab0ea97c476fbd4ffe288f2a4912cb06f1a47dcfa0 *ubuntu-18.04.3-desktop-amd64.iso" | hashit -H sha256 --check
```
or with detect

```sh
echo "add4614b6fe3bb8e7dddcaab0ea97c476fbd4ffe288f2a4912cb06f1a47dcfa0 *ubuntu-18.04.3-desktop-amd64.iso" | hashit -d --check
```

And then it should output it like this:
```sh
ubuntu-18.04.3-desktop-amd64.iso: OK
```

**Comment by cjavad** [comment]
Implemented as of acb922f

**Comment by cjavad** [comment]
Finally solved.
